Resources/Prototypes/Entities/Objects/Materials/materials.yml

```
@@ -63,6 +63,12 @@
     prototype: CarpetCard
     doAfter: 0.5
     removeOnInteract: true
+  # Moffstation - Start - Cardboard Module (please don't kill me Centronias)
```

```suggestion
  # Moffstation - Start - Cardboard Module
```
Comments which don't make sense outside of the PR shouldn't be included.
Ironic.

---

;-;